openmetadata-ui/src/main/resources/ui/src/components/Ingestion/Ingestion.component.tsx

```diff
@@ -113,7 +113,8 @@ const Ingestion: React.FC<IngestionProps> = ({
     if (ingestionList.length > 0) {
       return pipelineType.reduce((prev, curr) => {
         if (
-          curr !== PipelineType.Profiler &&
+          // Prevent adding multiple usage pipeline
```

Comment:
let's remove this now as it indicate the wrong scenario

Reply:
@ShaileshParmar11, That comment is part of this PR.